validphys2/src/validphys/scripts/vp_comparefits.py

```diff
@@ -76,6 +76,11 @@ def add_positional_arguments(self, parser):
             '--lite',
             help="Smaller version of the usual comparefit fit",
             action='store_true')
+        parser.add_argument(
+            '-p',
+            '--photon',
+            help="Add photon PDF to the report",
```

Comment:
```suggestion
            help="Use LUX basis (which include the photon) for the report",
```

to be a bit more specific on what the flag is doing.